cli/src/pixl_cli/_config.py

```diff
@@ -13,20 +13,57 @@
 #  limitations under the License.
 
 """Configuration of CLI from config file."""
-from pathlib import Path
+from typing import Optional
 
-import yaml
+from decouple import config
 
+API_SETTINGS = {"ehr_api": {}, "imaging_api": {}}  # type: dict
+API_SETTINGS["ehr_api"]["host"] = config("PIXL_EHR_API_HOST", default="localhost")
+API_SETTINGS["ehr_api"]["port"] = int(config("PIXL_EHR_API_PORT", default=7006))
+API_SETTINGS["ehr_api"]["default_rate"] = int(config("PIXL_EHR_API_RATE", default=1))
+API_SETTINGS["imaging_api"]["host"] = config("PIXL_IMAGING_API_HOST", default="localhost")
+API_SETTINGS["imaging_api"]["port"] = int(config("PIXL_IMAGING_API_PORT", default=7007))
+API_SETTINGS["imaging_api"]["default_rate"] = int(config("PIXL_IMAGING_API_RATE", default=1))
```

Comment:
If we're going to have this as a constant then we should initialise it all in one, so like this? May be a moot point if we create the API settings objects directly
```suggestion
API_SETTINGS = {
  "ehr_api": {
    "host": config("PIXL_EHR_API_HOST", default="localhost"),
    "port": int(config("PIXL_EHR_API_PORT", default=7006))
    "default_rate": int(config("PIXL_EHR_API_RATE", default=1))
    },
  }, 
  "imaging_api": {...}
  }

API_SETTINGS["imaging_api"]["host"] = config("PIXL_IMAGING_API_HOST", default="localhost")
API_SETTINGS["imaging_api"]["port"] = int(config("PIXL_IMAGING_API_PORT", default=7007))
API_SETTINGS["imaging_api"]["default_rate"] = int(config("PIXL_IMAGING_API_RATE", default=1))
```